source/sites/mon-entreprise.fr/pages/Stats/Stats.tsx

```diff
@@ -0,0 +1,389 @@
+			}}
+		/>
+		<div
+			css={`
+				font-weight: bold;
+				margin-left: 1rem;
+				color: var(--textColorOnWhite);
+			`}
+		>
+			<Value maximumFractionDigits={0} unit="visiteurs">
+				{visitors}
+			</Value>
+		</div>
+	</div>
+)
+let BranchIcône = ({ icône }) => (
+	<div className="distribution-chart__legend">
+		<span className="distribution-chart__icon">{emoji(icône)}</span>
+	</div>
+)
+
+type DistributionBranchProps = {
+	page: { label: string; nb_visits: number }
+	title: string
+	icon?: string
+	total: number
+}
+
+const Simulateurs = {
+	'/salarié': {
+		name: 'Salarié',
+		description:
+			"Calculer le salaire net, brut, ou total d'un salarié, stagiaire,ou assimilé",
+		icone: '🤝'
+	},
+	'/auto-entrepreneur': {
+		name: 'Auto-entrepreneur',
+		description:
+			"Calculer le revenu (ou le chiffre d'affaires) d'un auto-entrepreneur",
+		icone: '🚶‍♂️'
+	},
+	'/artiste-auteur': {
+		name: 'Artiste-auteur',
+		description: "Estimer les cotisations sociales d'un artiste ou auteur",
+		icone: '👩‍🎨'
+	},
+	'/indépendant': {
+		name: 'Indépendant',
+		description:
+			"Calculer le revenu d'un dirigeant de EURL, EI, ou SARL majoritaire",
+		icone: '👩‍🔧'
+	},
+	'/assimilé-salarié': {
+		name: 'Assimilé salarié',
+		description:
+			"Calculer le revenu d'un dirigeant de SAS, SASU ou SARL minoritaire",
+		icone: '☂️'
+	},
+	'/comparaison-régimes-sociaux': {
+		name: 'Comparaison status',
+		description:
+			'Simulez les différences entre les régimes (cotisations,retraite, maternité, maladie, etc.)',
+		icone: '📊'
+	}
+}
+
+export function DistributionBranch({
+	page,
+	title,
+	icon,
+	total
+}: DistributionBranchProps) {
+	const [intersectionRef, brancheInViewport] = useDisplayOnIntersecting({
+		threshold: 0.5
+	})
+	const { color } = useContext(ThemeColorsContext)
+	const visitors = brancheInViewport ? page.nb_visits : 0
+	const styles = useSpring({
+		config: ANIMATION_SPRING,
+		to: {
+			flex: visitors / total,
+			opacity: visitors ? 1 : 0
+		}
+	}) as { flex: number; opacity: number }
+
+	return (
+		<animated.div
+			ref={intersectionRef}
+			className="distribution-chart__item"
+			style={{ opacity: styles.opacity }}
+		>
+			<BranchIcône icône={icon} />
+			<div className="distribution-chart__item-content">
+				<p className="distribution-chart__counterparts">
+					<span className="distribution-chart__branche-name">{title}</span>
+					<br />
+				</p>
+				<ChartItemBar
+					{...{
+						styles,
+						color,
+						visitors,
+						total
+					}}
+				/>
+			</div>
+		</animated.div>
+	)
+}
+
+export default function Stats() {
+	const [choice, setChoice] = useState<LineChartVisitsProps['periodicity']>(
+		'monthly'
+	)
+	const [choicesimulators, setChoicesimulators] = useState('currentmonth')
+	const { color } = useContext(ThemeColorsContext)
+
+	return (
+		<>
+			<ScrollToTop />
+			<h1>
+				Statistiques <>{emoji('📊')}</>
+			</h1>
+			<p>
+				Découvrez nos statistiques d'utilisation mises à jour en temps réel. Les
```

Comment:
```suggestion
				Découvrez nos statistiques d'utilisation mises à jour quotidiennement. Les
```

(oui c'est moi qui ait écrit ça mais à la relecture ça me semble mieux)